frontend/model/contracts/group.js

```diff
@@ -239,15 +266,15 @@ DefineContract({
     'gi.contracts/group/invite': {
       validate: objectOf({
         inviteSecret: string, // NOTE: simulate the OP_KEY_* stuff for now
-        numInvites: number
+        numInvites: number,
+        invitee: string
       }),
       process (state, { data, meta }) {
-        Vue.set(state.invites, data.inviteSecret, {
-          generated: data.numInvites,
-          creator: meta.username,
-          responses: {},
-          status: 'valid'
-        })
+        Vue.set(state.invites, data.inviteSecret, initInvite({
+          inviteSecret: data.inviteSecret,
+          numInvites: data.numInvites,
+          creator: meta.username
+        }))
```

Comment:
missing `responses`, and `status`

Reply:
No need for that. When creating an invitation, it always has the same responses ans status values, so `initInvite` already does that for us ;)